host_vars/gub37-hof-n-5/base.yml

```diff
@@ -2,6 +2,6 @@
 
 location: gub37
 role: ap
-model: "ubnt_nanostation_loco_m5_xw"
+model: "mikrotik_sxtsq-5-ac"
 
-poe_on: []
\ No newline at end of file
+poe_on: []
```

Comment:
you can delete that line. it has no poe?

Reply:
Nice catch! Just pushed  a fix.